xml/admin_ceph_upgrade.xml

```diff
@@ -609,6 +609,13 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
      </para>
     </tip>
    </step>
+   <step>
+    <para>
+     Remove the node exporter. It does not need to be migrated and
+     will be installed as a container when the <filename>specs.yaml</filename> file is applied.
```

Comment:
Would "re-installed as a container" be equally true and perhaps a little more reassuring?